drivers/include/nrfx_nvmc.h

```diff
@@ -310,6 +324,17 @@ NRFX_STATIC_INLINE bool nrfx_nvmc_write_done_check(void)
     return nrf_nvmc_ready_check(NRF_NVMC);
 }
 
+NRFX_STATIC_INLINE uint32_t nrfx_nvmc_uicr_word_read(uint32_t const volatile *address)
+{
+	uint32_t value = *address;
```

Comment:
Since it is required that the address is word aligned, perhaps it would be good to add `NRFX_ASSERT(nrfx_is_word_aligned((void const *)address));` here.